pytest.ini

```diff
@@ -4,10 +4,10 @@ markers =
     tier2: Tier 2 cases for virt-who extra functions
     tier3: Tier 2 cases for virt-who extra functions
 
-    satellite: cases for satellite
-    rhsm: cases for candlepin
+    notSatellite: cases not for satellite
+    notRHSM: cases not for candlepin
 
-    rhel8: cases only for rhel 8
-    rhel9: cases only for rhel 9
+    notRHEL8: cases not for rhel8
+    notRHEL9: cases not for rhel9
 
```

Comment:
All cases need to update this...

Reply:
yes, and I think we can maintain the marks in cases at last in another single pr.